driftbase/parties.py

```diff
@@ -22,6 +22,7 @@
 
 
 def accept_party_invite(invite_id, sending_player, accepting_player):
+    max_players_per_party = 4 # FIXME: Probably belongs in the product config
```

Comment:
Correct, and this can somewhat easily be added, here's some code fetching an S3 bucket name from config:

```
    tenant = g.conf.tenant
    if tenant and 'assets' in tenant:
        assets_config = tenant['assets']
        bucket = assets_config.get('asset_bucket_name')
        return bucket
```

This can then just be added manually to the config, or one can add an actual resource which configures defaults and adds an entry to the config as it's registered. See the `resources` folder in drift, drift-base, etc.

If there's no default in the config, the code should return a default, as the config might not have been updated.